app/controllers/bedtime_stories.controller.js

```diff
@@ -171,3 +171,70 @@ BedtimeStory.findByPk(id,{
   });
 };
 
+// Update a Bedtime Story by the id in the request
+exports.updateStory = (req, res) => {
+  const id = req.params.id;
+
+  BedtimeStory.update(req.body, {
+    where: { story_id: id },
+  })
+    .then((num) => {
+      if (num == 1) {
+        res.send({
+          message: "Bedtime Story was updated successfully.",
+        });
+      } else {
+        res.send({
```

Comment:
should `console.log` the problem before sending a response both here and below on line 193, so there is a clear record of what went wrong on the backend logs rather than assuming the frontend will log such information